lib/decidim/term_customizer/import/readers/csv.rb

```diff
@@ -12,7 +12,7 @@ class CSV < Base
           MIME_TYPE = "text/csv"
 
           def read_rows
-            ::CSV.read(file, col_sep: ";").each_with_index do |row, index|
+            ::CSV.read(file, col_sep: TermCustomizer.csv_col_sep).each_with_index do |row, index|
```

Comment:
Could you just use the Decidim config variable? I don't think we need own configuration for Term Customizer.
```suggestion
            ::CSV.read(file, col_sep: Decidim.default_csv_col_sep).each_with_index do |row, index|
```

Please also remove the related `config_accessor` from the module.

Reply:
Sorry for the delay in checking, thanks for the feedback!  merge this fix.